boards/arm/stm32h747i_disco/Kconfig.defconfig

```diff
@@ -38,6 +38,13 @@ config CLOCK_STM32_D3PPRE
 config STM32H7_DUAL_CORE
 	default y
 
+if DISK_ACCESS
+
+config DISK_ACCESS_STM32_SDMMC
```

Comment:
This is not needed: this symbol is defined in `subsys/disk/Kconfig` as follows:
```
DT_COMPAT_ST_STM32_SDMMC := st,stm32-sdmmc
[...]
if DISK_ACCESS
[...]
config DISK_ACCESS_STM32_SDMMC
	bool "STM32 SDMMC driver"
	depends on HAS_STM32CUBE
	select USE_STM32_HAL_SD
	select USE_STM32_LL_SDMMC
	default $(dt_compat_enabled,$(DT_COMPAT_ST_STM32_SDMMC))
	help
	  File system on sdmmc accessed through stm32 sdmmc.
[...]
endif # DISK_ACCESS
```

This means it will be set to 'y' if `CONFIG_DISK_ACCESS=y` and a node with compat `st,stm32-sdmmc` is enabled.

Reply:
Removing this would fix CI issue.

Reply:
Changed, sample now works